src/js/button.js

```diff
@@ -1,7 +1,9 @@
 import Component from './component';
-import * as Lib from './lib';
+import * as Dom from './utils/dom.js';
 import * as Events from './events';
+import * as Fn from './utils/fn.js';
 import document from 'global/document';
+import assign from 'lodash.assign';
```

Comment:
I guess we'd want to do something like:
```js
import objectAssign from 'object.assign';
let assign = Object.assign || objectAssign;
```
As it is now, it isn't bad, but we'll always be using the polyfill and never the native.